src/Umbraco.Cms.Integrations.Search.Algolia/Services/IScopeService.cs

```diff
@@ -0,0 +1,17 @@
+﻿
+namespace Umbraco.Cms.Integrations.Search.Algolia.Services
+{
+    public interface IScopeService<T>
```

Comment:
This seems to be very generic - in the C# and English sense! - but it only seems to be used in one way.  Might be better just removing the generic parameter, and use the specific type.  And rename to something like `IAlgoliaIndexDefinitionStorage` (or similar)?